Lab_1 (1).ipynb

```diff
@@ -0,0 +1,948 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b02e9f7-a0a7-46a7-9b01-0638a1024f17",
+   "metadata": {},
+   "source": [
+    "This is another method to get the same answer as above, for 2 sigma. Since the normal distribution is symmetric we can integrate up to -2 sigma, which would be 2 sigma to the left of the mean, a much smaller probability. If we calculate this percentage, we must subtract it from 1 in order to get the same answer for 2 sigma. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "8b17eba2-4d32-491a-a41b-c21900f31186",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.9772498680518208\n"
+     ]
+    }
+   ],
+   "source": [
+    "prob_neg2 = stats.norm.cdf(-2, loc = 0, scale = 1)\n",
+    "print(1-prob_neg2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9b55b904-b7a0-4a26-ae1f-9f9677b63ce7",
+   "metadata": {},
+   "source": [
+    "Probability of sigma = 5, up to 5 sigma is going to be very close to 100% because it contains almost the entire area under the normal distribution."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "9899f921-ccd5-4dfb-9999-ef988e9a81fb",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.9999997133484281\n"
+     ]
+    }
+   ],
+   "source": [
+    "prob_5 = stats.norm.cdf(5, loc = 0, scale = 1)\n",
+    "print(prob_5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d387d47-97c0-4c7a-a6a6-66d075302f58",
+   "metadata": {},
+   "source": [
+    "What if we only want the probability of 2 sigma or higher? There are 2 ways to approach this: We could do the integral up to -2 sigma, or we could subtract the integral up to 2 sigma from 1."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "2ed5dc93-8ca4-4bab-9d47-ef9e77f90ea3",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.022750131948179195\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(prob_neg2)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "id": "7cb30467-9513-44f1-ab0b-45b3619503f9",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0.02275013194817921\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(1-prob_2)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0d420b0c-eff5-4813-84f3-ddf8042f59d4",
+   "metadata": {},
+   "source": [
+    "# Now we can go the opposite way using the percent point function stats.norm.ppf(), which is simply the inverse of the cdf(). With this function we can input a percentage probability and find the sigma value that this corresponds to. Again, this will provide the answer as if we were integrating up to the value from negative infinity, not from the mean. "
```

Comment:
This should be a lower heading, it's so big and bold it's difficult to read due to its length